js/glossary.js

```diff
@@ -1,11 +1,6 @@
----
-layout: null
----
 var glossary = [
-{% for entry in site.data.glossary %}
-{
-"term": {{ entry[0] | jsonify }},
-"def": {{ entry[1] | markdownify | replace:'href="#','href="/glossary/?term=' | jsonify }}
-}{% unless forloop.last %},{% endunless %}
-{% endfor %}
+{%- for entry in site.data.glossary -%}
+    {"term": {{ entry[0] | jsonify }}, "def": {{ entry[1] | markdownify | replace:'href="#','href="/glossary/?term=' | jsonify }}}
```

Comment:
We probably don't need to replace the `href="/glossary/?term=`. There's some old script that used this, but looks like it's no longer used. Will have to review that, and can remove in a follow-up

Reply:
Follow-up in https://github.com/docker/docker.github.io/pull/11499